app/main.cpp

```diff
@@ -393,6 +392,7 @@ int main( int argc, char *argv[] )
   InputTests tests;
   tests.parseArgs( argc, argv );
 #endif
+  qDebug() << "Mergin Maps Input App" << version << InputUtils::appPlatform() << "(" << CoreUtils::appVersionCode() << ")";
   qDebug() << "Built with QGIS version " << VERSION_INT;
```

Comment:
Maybe both of these could be `CoreUtils::log` instead of `qDebug`?

Reply:
I am not sure, in log we have it in separate entry... I think here it is more useful for developers on top of the application output to see if they are compiling with correct QT and App version...